ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.10.yaml

```diff
@@ -75,19 +75,9 @@ tests:
   steps:
     cluster_profile: openstack-vh-mecha-az0
     env:
-      TEST_SKIPS: Kubectl Port forwarding With a server listening on 0.0.0.0 should
-        support forwarding over websockets \| Kubectl client Simple pod should support
-        exec through kubectl proxy \| Kubectl client Simple pod should support exec
-        through an HTTP proxy \| Kubectl Port forwarding With a server listening on
-        localhost should support forwarding over websockets \| Pods should support
-        remote command execution over websockets \| Pods should support retrieving
-        logs from the container over websockets \| NetworkPolicy between server and
-        client should deny ingress access to updated pod \| The HAProxy router should
-        be able to connect to a service that is idled because a GET on the route will
-        unidle it \| Image append should create images by appending them \| deploymentconfigs
-        when run iteratively should only deploy the last deployment \| In-tree Volumes
+      TEST_SKIPS: In-tree Volumes
```

Comment:
Are all the other tests skipped in the origin repo?

Reply:
Apparently yeah.
I'm going to do a /retest, to see if the errors on 4.10 and 4.9 are consistent.
Thanks for holding it.